client/src/components/CommentApp/main.tsx

```diff
@@ -354,6 +354,15 @@ export class CommentApp {
         }
       }
     });
+
+    document.body.addEventListener('visibilityChange', () => {
```

Comment:
Could we name this something like `commentAnchorVisibilityChange` so people know it needs to be fired when comment anchors change?

Reply:
Yep, can do - we can rename it more generically when we actually use it for something else